composer.json

```diff
@@ -4,8 +4,8 @@
     "type": "wordpress-plugin",
 	"license": "GPL-2.0-or-later",
     "require": {
-        "facebook/graph-sdk": "^5.7",
-		"abraham/twitteroauth": "^1.0"
+        "nickdnk/graph-sdk": "^7.0.1",
+		"abraham/twitteroauth": "^4.0.1"
```

Comment:
@shiprahm nitpick: can you make this align?

Reply:
It's aligned already, probably a github alignment thing. See this is how it looks in code - 

<img width="887" alt="Screenshot 2022-11-17 at 11 33 11 AM" src="https://user-images.githubusercontent.com/110240621/202368914-79f53218-3b1f-4a4e-9fd7-40cebbb2e7e0.png">

Reply:
yes, that was due to extra whitespaces, corrected the same.